src/Core/Extensions/MessageExtensions.cs

```diff
@@ -14,22 +14,40 @@ public static async Task SendToChannel(this Embed e, IMessageChannel channel)
             => await channel.SendMessageAsync(string.Empty, false, e);
         public static EmbedBuilder AddFieldConditional(this EmbedBuilder eb, bool condition, string name, object value, bool inline = false)
         {
-            if (condition)
-                eb.AddField(name, value, inline);
+            if (condition) {
+                if (value is string) {
```

Comment:
Maybe convert `value` into a string in general, and then crop that, just incase the `.ToString()` method ends up being longer than the limit.